assets/js/modules/search-console/components/dashboard/SearchFunnelWidget/Overview.js

```diff
@@ -158,33 +174,49 @@ const Overview = ( {
 	return (
 		<Grid>
 			<Row>
-				<Cell { ...quarterCellProps }>
-					<DataBlock
-						stat={ 0 }
-						className="googlesitekit-data-block--impressions googlesitekit-data-block--button-1"
-						title={ __( 'Total Impressions', 'google-site-kit' ) }
-						datapoint={ totalImpressions }
-						change={ totalImpressionsChange }
-						changeDataUnit="%"
-						context="button"
-						selected={ selectedStats === 0 }
-						handleStatSelection={ handleStatsSelection }
-					/>
-				</Cell>
+				{ isSearchConsoleGatheringData && ! hasSearchConsoleData && (
```

Comment:
This condition can be simplified as it isn't possible for the module to be gathering data and have data. Gathering data means there is no data available yet.
```suggestion
				{ isSearchConsoleGatheringData && (
```